getting-started/telerik-nuget-server.md

```diff
@@ -21,9 +21,9 @@ In the the Package Sources section users can add new sources.
 
 ![](images/nuget-server/nuget-vs-add-source.png)
 
-In the Source field users should fill in the address of the Telerik server (URL: **https://nuget.telerik.com/nuget**) and click the Update button.
+In the Source field users should fill in the address of the Telerik server (URL: **https://nuget.telerik.com/v3/index.json**) and click the Update button.
 
->tip The improved Telerik NuGet v3 is now available for beta testing at [https://nuget.telerik.com/v3/index.json](https://nuget.telerik.com/v3/index.json). The new v3 API is faster, lighter, and reduces the number of requests from NuGet clients. You are welcome to try it.
+>important The previous version of the Telerik NuGet server (**https://nuget.telerik.com/nuget**) will be deprecated soon. Update your settings to point to the new v3 API ((URL: **https://nuget.telerik.com/v3/index.json**)), which is faster, lighter, and reduces the number of requests from NuGet clients. 
```

Comment:
I would remove "soon":
The previous version of the Telerik NuGet server (**https://nuget.telerik.com/nuget**) will be deprecated.